### ISSUE TITLE
responseStatusCode 200

### ISSUE THREAD

**Issue by VA-PLE** [projectcontour/contour#3880]
**Please describe the problem you have**
Hello. Is it possible to receive 200-204 responseStatusCode? I see the boundaries now 400-599. Сan I somehow get around it? Maybe there are examples.

**Comment by youngnick** [comment]
Hi @VA-PLE, sorry but I'm unclear what you are asking for here. Do you mean that you would like ratelimited requests to be sent with a 200? That seems like a strange behavior, since the request was not successful if it's rate-limited. Could you explain more?

**Comment by VA-PLE** [comment]
Yes. Our customers expect 200 code when everything goes well and 204 in all other cases

**Comment by VA-PLE** [comment]
I experimented with envoy. Everything is simple there

**Comment by youngnick** [comment]
Sorry @VA-PLE, it seems like you want Contour and Envoy to translate 503 responses from backends into 204 ones? That seems pretty counter to the HTTP specs to me.

I also see you've closed this, if you would like to continue here, please comment and I will reopen. If not, thanks for the issue.
